plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/maven/MavenArtifact.java

```diff
@@ -145,31 +156,58 @@ private void parseMetadata(InputStream mdStream) throws IOException, XMLExceptio
         SAXReader reader = new SAXReader(mdStream);
         reader.parse(new SAXListener() {
             public String lastTag;
+            public boolean insideSnapshotVersion = false;
+            public String currentSnapshotVersion;
+            boolean invalid = false;
 
             @Override
             public void saxStartElement(SAXReader reader, String namespaceURI, String localName, Attributes atts) throws XMLException {
+                if ("snapshotVersion".equals(localName)) {
```

Comment:
Please create a constant for "snapshotVersion". You used it many times in one file.